devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy

```diff
@@ -237,3 +239,16 @@ def configure() {
 	// battery minReport 30 seconds, maxReportTime 6 hrs by default
 	return refresh() + zigbee.batteryConfig() + zigbee.temperatureConfig(30, 300) // send refresh cmds as part of config
 }
+
+private isFirmwareVerLessThan(major, minor, build) {
+	def deviceFwVer = "${device.getFirmwareVersion()}"
```

Comment:
I wish there was a better interface to get at the firmware version but it looks like this is the best we can do for now because we can't call functions or access members on the object returned by `getFirmwareVersion()`

Reply:
I agree. I doubt the DeviceFirmware toString will change, but this should be "ok" for now.